### ISSUE TITLE
repo name

### ISSUE THREAD

**Issue by nsheff** [cphg/scribe#1]
Would it be better to call this `compgen-scribe` to tie it to that course? Or do you think this will will eventually be used for all courses?

**Comment by aakrosh** [comment]
Good point. I think we can change the name to compgen-scribe to reflect that this is the scribe for that course, unless there are any other objections. We can also change the name of the repository later, if we find that it conflicts with scribe that we want for another course.
